org.srb2.SRB2Kart.json

```diff
@@ -0,0 +1,99 @@
+                    "sha256": "aba34e53ef0ec6a34b58b84e28bf8cfbccee6585cebca25333604c35db3e051d"
+                }
+            ]
+        },
+        {
+            "name": "srb2kart-assests",
+            "disabled": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "7za x srb2kart-v12-Installer.exe",
+                "mkdir -p /app/usr/share/games/SRB2Kart",
+                "install -Dm644 {music,textures,gfx,maps,sounds,chars,bonuschars}.kart srb2.srb /app/usr/share/games/SRB2Kart",
+                /* support for patching, not needed for v1.2 */
+                "# unzip -o srb2kart-v12-Patch.zip",
+                "# install -Dm664 patch.kart /app/usr/share/games/SRB2Kart"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/STJr/Kart-Public/releases/download/v1.2/srb2kart-v12-Installer.exe",
+                    "sha256": "57284de43a2889c0a60ce8fee77df22095fcd31beef6130cbeed1ead0bebbfec"
+                }
+            ]
+        },
+        {
+            "name": "srb2kart",
+            "buildsystem": "cmake",
+            "config-opts": ["-DGME_INCLUDE_DIR=/app/include/gme/"],
+            "builddir": true,
+            "build-commands": [
+                "ln -s /app/srb2kart /app/bin/",  /* why did it put it there? */
+                "install -Dm644 ../src/sdl/SDL_icon.xpm /app/usr/share/pixmaps/srb2kart.xpm",
+                "install -Dm644 ../org.srb2.SRB2Kart.appdata.xml /app/usr/share/metainfo",
+                "install -Dm644 ../org.srb2.SRB2Kart.desktop /app/usr/share/applications"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/STJr/Kart-Public/archive/v1.2.zip",
+                    "sha256": "3016627dab91590dea7075d9794de87412db5c9e9de2abb6c1dd943eca1bef5c"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/STJr/Kart-Public/releases/download/v1.2/srb2kart-v12-Installer.exe",
+                    "sha256": "57284de43a2889c0a60ce8fee77df22095fcd31beef6130cbeed1ead0bebbfec"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "mkdir tmp",
+                        "cd tmp; 7za x ../srb2kart-v12-Installer.exe",
+                        "mkdir -p assets/installer/",
+                        "cp tmp/srb2.srb assets/installer/",
+                        "cp tmp/{patch,gfx,textures,chars,maps}.kart assets/installer/",
+                        "cp tmp/{music,sounds,bonuschars}.kart assets/installer/"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "org.srb2.SRB2Kart.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html